test/Microsoft.PowerShell.SecretManagement.Tests.ps1

```diff
@@ -1,52 +1,39 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-Describe "Test Microsoft.PowerShell.SecretManagement module" -tags CI {
+Describe "Test Microsoft.PowerShell.SecretManagement module" {
+    BeforeDiscovery {
+        $TestCases = 'ByteArray', 'String', 'SecureString', 'PSCredential', 'Hashtable'
+    }
 
     BeforeAll {
+        $ProjectRoot = Split-Path $PSScriptRoot
+        $ModulePath = Join-Path $ProjectRoot "module"
+        $ManifestPath = Join-Path $ModulePath 'Microsoft.PowerShell.SecretManagement.psd1'
 
-        if ((Get-Module -Name Microsoft.PowerShell.SecretManagement -ErrorAction Ignore) -eq $null)
+        $BasePath = Join-Path ([IO.Path]::GetTempPath()) "SecretManagementStorePath"
```

Comment:
rather than `GetTempPath` perhaps you could use `$TESTDRIVE` which is cleaned up by pester when the `Describe` exits

Reply:
Nice!